commons-math-legacy/src/main/java/org/apache/commons/math4/legacy/fitting/leastsquares/LevenbergMarquardtOptimizer.java

```diff
@@ -738,9 +736,9 @@ private double determineLMParameter(double[] qy, double delta, double[] diag,
             // if the function is small enough, accept the current value
             // of lmPar, also test for the exceptional cases where parl is zero
             if (JdkMath.abs(fp) <= 0.1 * delta ||
-                (parl == 0 &&
+                 parl == 0 &&
```

Comment:
Now you have fixed the checkstyle suppressions file, you can revert the changes to the parentheses